Source/WebCore/rendering/RenderVideo.cpp

```diff
@@ -29,6 +29,7 @@
 #include "RenderVideo.h"
```

Comment:
The changes in RenderVideo.h/cpp are to fix the same issue 265672@main fixed but for fullscreen video.

They can be broken out into a separate patch if preferred, the issues are independent.